restbridge/RestBridge.cpp

```diff
@@ -33,128 +34,126 @@
 
 namespace restbridge
 {
-
-static const std::string PUBLISHER_SCHEMA_SUFFIX = "cmd://";
-static const std::string SUBSCRIBER_SCHEMA_SUFFIX = "resp://";
+namespace
```

Comment:
http://stackoverflow.com/questions/4422507/superiority-of-unnamed-namespace-over-static